src/Components/Container/Container.js

```diff
@@ -3,12 +3,14 @@ import { connect } from 'react-redux';
 import Dimensions from 'Dimensions';
 import { Text, View, StyleSheet, Image } from 'react-native';
 import { Card, Icon, ListItem, Button } from 'react-native-elements';
+// 
```

Comment:
Glad you told me why you separate the package imports from the codebase imports.  Good for dev empathy.